src/features/embalm/stepContent/components/ArchaeologistListItem.tsx

```diff
@@ -82,62 +82,69 @@ export function ArchaeologistListItem({
     );
   }
 
-  const handleClickRow = () => {
-    onClick();
-  };
-
   return (
-    <Tr
-      background={isSelected ? (archaeologist.exception ? 'background.red' : 'brand.50') : ''}
-      onClick={() => handleClickRow()}
-      cursor="pointer"
-      _hover={isSelected ? {} : { background: 'brand.0' }}
+    <Tooltip
```

Comment:
This component needs to return a `Tr` element since it's a direct child of `Tbody`

Reply:
Doesn't throw an error, just the following warning: 

```
react-dom.development.js:84 Warning: validateDOMNesting(...): <span> cannot appear as a child of <tbody>.
```

Reply:
Gotcha